src/core/send.c

```diff
@@ -118,7 +118,7 @@ _IRQL_requires_max_(DISPATCH_LEVEL)
 void
 QuicSendQueueFlush(
     _In_ QUIC_SEND* Send,
-    _In_ QUIC_SEND_FLUSH_REASON Reason
+    _In_ QUIC_SEND_FLUSH_REASON Reason // NOLINT
```

Comment:
I prefer `UNREFERENCED_PARAMETER` instead of `NOLINT` for variables only used in tracing.

Reply:
See above. Am fine with this, but I felt it was confusing, because the variable is referenced when the trace call isn't a no-op.

Reply:
`// NOLINT` isn't going to help cross-platform if logging is disabled, but `UNREFERENCED_PARAMETER` will. BTW, I don't want to worry/care/optimize-for the no-logging scenario since it's not the norm.